main.js

```diff
@@ -256,9 +285,20 @@ $cartSummaryView.addEventListener('click', (event) => {
     app.view = 'catalog'
     renderApp(app)
   }
+  else if (event.target.id === 'cart-checkout') {
+    app.view = 'checkout'
+    renderApp(app)
+  }
 })
 
 $cart.addEventListener('click', (event) => {
   app.view = 'cart'
+  app.cart.total = calculateCartTotal(app.cart)
   renderApp(app)
 })
+
+$checkoutView.addEventListener('click', (event) => {
```

Comment:
In general, we do not listen for `click` events when dealing with forms. Instead, handle a `submit` event. But be sure to prevent the default behavior of form submissions.